src/main/java/logging/Utilities.java

```diff
@@ -0,0 +1,16 @@
+package logging;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Utilities {
+	public final Logger logger = LoggerFactory.getLogger( "AnyUniqueStringHere" );
```

Comment:
remove this class from the project. we disc ussed to use logger directly inside every class where it is required.